content/en/user-guide/aws/glue/index.md

```diff
@@ -4,13 +4,29 @@ linkTitle: Glue
 description: Get started with Glue on LocalStack
 tags: ["Pro image"]
 ---
+
 ## Introduction
 
 The Glue API in LocalStack Pro allows you to run ETL (Extract-Transform-Load) jobs locally, maintaining table metadata in the local Glue data catalog, and using the Spark ecosystem (PySpark/Scala) to run data processing workflows.
 
 LocalStack allows  you to use the Glue APIs in your local environment.
 The supported APIs are available on our [API coverage page](/references/coverage/coverage_glue/), which provides information on the extent of Glue's integration with LocalStack.
 
+{{< callout >}}
+LocalStack now includes a container-based Glue Job executor, allowing Glue jobs to run inside a Docker environment.
+Previously, LocalStack used a pre-packaged binary with Spark and required components.
+The new executor uses the `aws-glue-libs` Docker image, offering better production parity, faster startup, and more reliable execution.
+
+It adds support for:
+
```

Comment:
we can also mention with the new executor we are able to run glue job in isolation from other jobs